varia/style.css

```diff
@@ -140,6 +140,12 @@ $grid-configuration: map-extend($grid-configuration-default, $grid-configuration
  * Add font-family using CSS variables.
  * It also adds the proper fallback for browsers without support.
  */
+body {
+	display: none;
+	/* Legacy */
+	display: var(--skip, block);
```

Comment:
Should/could this go in Varia?

Reply:
it's in Varia, haha

Reply:
Oh so it is. I guess we'll need to rebuild all child themes then...